web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md

```diff
@@ -0,0 +1,148 @@
+| 10:20 - 10:35 |    | **COFFEE BREAK** |  |
+| 10:35 - 11:20 | 4  | Roadmap: Simulations Development |  |
+| 11:20 - 12:05 | 5  | Roadmap: Experimental Diagnostics Development |  |
+| 12:05 - 13:30 |    | **LUNCH BREAK** |  |
+| 13:30 - 14:15 | 6  | Roadmap: Development of Remaining Elements |  |
+| 14:15 - 15:00 | 7  | Community Interaction & Contribution |  |
+| 15:00 - 15:15 |    | **COFFEE BREAK** |  |
+| 15:15 - 16:00 | 8  | Education & Outreach |  |
+| 16:00 - 16:45 | 9  | Student Projects |  |
+| 16:45 - 17:30 | 10 | Open Discussion for Uncovered Topics |  |
+| 17:30 - 18:00 | 11 | Summarize Tasks & Responsibilities |  |
+|               |    | **ADJOURN** |  |
+
+---
+_**Day 2 - April 21, 2020**_
+
+Day 2 is the first day for the broader PlasmaPy Community.  The day will be divided into
+three parts, (1) introducing/sharing PlasmaPy's plans to the community, (2) demonstrating
+examples of how the code works, and (3) conversing with the community to get feedback
+and suggestions.  This day will also be available via [web-cast](#Web-Participation).
+
+| Time | Section | Title | Speaker/Leader |
+| --- | --- | --- | --- |
+| 08:30 - 09:00 | 0  | **ARRIVAL & COFFEE** |  |
+| 09:00 - 09:10 | 1  | Welcome to Users |  |
+| 09:10 - 10:00 | 2  | PlasmaPy Overview |  |
+| 10:00 - 10:20 | 3  | PlasmaPy - Simulations |  |
+| 10:20 - 10:35 |    | **COFFEE BREAK** |  |
+| 10:35 - 10:55 | 4  | PlasmaPy - Experimental Diagnostics |  |
+| 10:55 - 11:45 | 5  | Community Interaction |  |
+| 11:45 - 13:15 |    | **LUNCH BREAK** |  |
+| 13:15 - 13:35 | 6  | Education & Outreach |  |
+| 13:35 - 13:55 | 7  | Contributing to PlasmaPy |  |
+| 13:55 - 14:30 | 8  | Demo - Working through a Pull Request |  |
+| 14:30 - 14:50 | 9  | Demo - #2 |  |
+| 14:50 - 15:05 |    | **COFFEE BREAK** |  |
+| 15:05 - 15:25 | 10 | Demo - #3 |  |
+| 15:25 - 17:00 | 11 | Community Discussion |  |
+|               |    | **ADJOURN** |  |
+
+---
+_**Day 3 - April 22, 2020**_
+
+Day 3 is a workshop day to provide hands-on experience with PlasmaPy and one-on-one 
+community interaction.  In the morning session we will provide a few mini 
+example projects to highlight PlasmaPy's features.  The afternoon session will
+be an open session where users can interact directly with the PlasmaPy team, asking
+an relevant questions, getting help on existing projects, etc.
+
+| Time | Section | Title | Speaker/Leader |
+| --- | --- | --- | --- |
+| 08:30 - 09:00 | 0  | **ARRIVAL & COFFEE** |  |
+| 09:00 - 12:00 | 1  | Workshop: Examples Projects |  |
+| 12:00 - 13:30 |    | **LUNCH BREAK** |  |
+| 13:30 - 16:00 | 4  | Open Session |  |
+|               |    | **ADJOURN** |  |
+
+<br/>
+
+### <a name="Travel-Housing"></a> Travel & Housing
+
+---
+**NOTE**
+
+Due to the ongoing development of the novel coronavirus (COVID-19), travel restrictions
+may be in place when this workshop occurs.  If this is the case, we will still proceed 
+with the workshop via a [web-cast](#Web-Participation).
```

Comment:
Here we should recommend strongly against purchasing nonrefundable plane tickets.